website/content/en/v1.2/concepts/disruption.md

```diff
@@ -358,6 +358,22 @@ spec:
         karpenter.sh/do-not-disrupt: "true"
 ```
 
+Pods with PDBs will not be gracefully evicted by the [Termination Controller]({{<ref "#termination-controller">}}) or be considerd for consolidation. When multiple pods on a node have different PDBs, all PDB constraints must be satisfied simultaneously for Karpenter to consolidate the node. This can create complex eviction scenarios:
+  - If a pod matches multiple PDBs (via label selectors), ALL of these PDBs must allow for disruption
+  - When different pods on the same node belong to different PDBs, ALL PDBs must simultaneously permit eviction
+  - A single blocking PDB can prevent the entire node from being consolidated
+
+For example, consider a node with these pods and PDBs:
+- Pod A: Matches PDB-1 (maxUnavailable: 0) and PDB-2 (maxUnavailable: 1)
+- Pod B: Matches PDB-3 (minAvailable: 100%)
+- Pod C: No PDB
+
+In this scenario, Karpenter cannot consolidate the node because:
+1. Pod A is blocked by PDB-1 even though PDB-2 would allow disruption
+2. Pod B is blocked by PDB-3's requirement for 100% availability
+
+Karpenter will not be able to complete consolidation and termination of the node until all blocking PDBs allow for pod eviction or the pods are removed through other means.
```

Comment:
nit: I think this is understood implicitly and muddies the waters (e.g. a user wonders what other means Karpenter has to remove a pod). I think we can just end this section off with a summary stating that the more PDBs there are affecting a node, the more difficult it will be for Karpenter to perform voluntary disruption.

Also, I didn't leave this comment elsewhere but we should talk about voluntary disruption generally, rather than just consolidation.

```suggestion
As seen in this example, the more PDBs there are affecting a Node, the more difficult it will be for Karpenter to find an opportunity to perform voluntary disruption actions. 
```